client/src/pages/Rides.js

```diff
@@ -7,6 +7,9 @@ import Card from "../components/Card";
 import HeaderMain from "../components/HeaderMain";
 import ButtonPlus from "../components/ButtonPlus";
 import { useHistory } from "react-router-dom";
+import { getCurrentDate } from "../utils/date";
+import { getDataByQuery } from "../utils/api";
+import { useEffect } from "react";
```

Comment:
Sort your imports according to importance e.g. React at the top. Feel free to ignore this as this is purely code style preference

Reply:
Sorted them, thanks for the advice 😄 🚀